### PR TITLE
created gamefi page

### PR DESCRIPTION
**Pull request by rarcif**

### CODE REVIEW

File: docs/getting-started/gamefi-sdk.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+string rpc = "https://evm-cronos.crypto.org";
+
+string balance = await EVM.BalanceOf(chain, network, account, rpc);
+print(balance);
+```
+
+Verify a signed message.
+```csharp
+string message = "YOUR_MESSAGE";
+string signature = "0x94bdbebbd0180195b89721a55c3a436a194358c9b3c4eafd22484085563ff55e49a4552904266a5b56662b280757f6aad3b2ab91509daceef4e5b3016afd34781b";
```
Comment (lezzokafka) [inline docs/getting-started/gamefi-sdk.md:318]: Maybe change the sample signature to some string to be filled?

**Review by lezzokafka — COMMENTED** [review]
Thanks @rarcif, left some quick comments.

File: docs/getting-started/gamefi-sdk.md (hunk 1 above)
Comment (rarcif) [reply docs/getting-started/gamefi-sdk.md]: done :)

**Pull request merged**